Payload/TaxonPayload.php

```diff
@@ -78,6 +79,14 @@ public function getTranslations(): array
         return $translations;
     }
 
+    /**
+     * @return mixed[]
+     */
+    public function getImages(): array
```

Comment:
not tested

Reply:
?? a test which tests the exception ?? i am not sure if that makes sense 🙈